src/main/java/entityController/AddressController.java

```diff
@@ -0,0 +1,142 @@
+package entityController;
+
+import entity.Address;
+import entity.Employee;
+import listEntity.AddressItem;
+import service.AddressService;
+import service.EmployeeService;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import static validator.InputValue.*;
+import static validator.validationCountry.getNameCountry;
+
+public class AddressController {
+
+    AddressService addressService = new AddressService();
+
+    public Address inputAddress() throws SQLException {
+        Address address = new Address();
+        System.out.println("Enter country");
+        String country = getNameCountry();
+        address.setCountry(country);
+        System.out.println("Enter town");
+        String town = inputValidateStr();
+        address.setTown(town);
+        System.out.println("street");
+        String street = inputValidateStr();
+        address.setStreet(street);
+        System.out.println("postCode");
+        String postCode = inputPostCodeValue();
+        address.setPostCode(postCode);
```

Comment:
better to collect all inputs and create an address object with a constructor or builder